tests/storage/migration.go

```diff
@@ -539,6 +539,23 @@ var _ = SIGDescribe("[Serial]Volumes update with migration", Serial, func() {
 			Eventually(ThisVMIWith(vm.Namespace, vm.Name), 120*time.Second, 1*time.Second).Should(BeRunning())
 		})
 
+		It("should set the ManualRecoveryRequired field and return an error if the VM is restarted during the volume migration", func() {
+			volName := "volume"
+			dv := createDV()
+			vm := createVMWithDV(dv, volName)
+			// Create dest PVC
+			createUnschedulablePVC(destPVC, ns, size)
+			By("Update volumes")
+			updateVMWithPVC(vm, volName, destPVC)
+			waitMigrationToExist(vm.Name, ns)
+			waitVMIToHaveVolumeChangeCond(vm.Name, ns)
+
+			By("Restarting the VM during the volume migration")
+			restartOptions := &virtv1.RestartOptions{GracePeriodSeconds: pointer.P(int64(0))}
+			err := virtClient.VirtualMachine(vm.Namespace).Restart(context.Background(), vm.Name, restartOptions)
+			Expect(err).To(MatchError(ContainSubstring("VM recovery required")))
+		})
```

Comment:
Can't this be merged with the other one for start?
I mean, first try to restart, ensure that the request is blocked, then stop and try to start